ebl/v3/EBL_v3.0.0.yaml

```diff
@@ -6114,9 +6385,33 @@ components:
             - `DDS` (Consignee's freight forwarder)
             - `COW` (Invoice payer on behalf of the consignor (shipper))
             - `COX` (Invoice payer on behalf of the consignee)
-            - `N1` (First Notify Party)
-            - `N2` (Second Notify Party)
-            - `NI` (Other Notify Party)
+            - `CSR` (Consolidator)
+            - `MFR` (Manufacturer)
+            - `WHK` (Warehouse Keeper)
+          example: DDS
+      required:
+        - party
+        - partyFunction
+
+    OtherDocumentPartyHBoL:
```

Comment:
Do we need it as HBoL? as in the isHBLIssued it is just HBL, don't we have to make House Bill of Lading in attribute name consistent?